.flake8

```diff
@@ -0,0 +1,24 @@
+[flake8]
+max-line-length=127
```

Comment:
would vote 120 (simply because idies uses such limit elsewhere), but not a strong vote :smile:

Reply:
I'll address this in #10.